ros_topic:
  manual_cmd: "/hw/manual_cmd"
  auto_cmd: "/hw/autonomous_cmd"
  cmd_info: "/hw/info"
  emergency: "/hw/emergency_brake"
  autonomous_mode: "/autonomous_driver"
  cmd_sim: "/hw/cmd_sent"

sim_publisher: true

min_msg_period: 0.2 # [s] if msg have period greater than this the car is forced to a neutral actuator position

max_steering: 25 # [degree]

steer_pwm:
  center: 1.5
  range: 0.37

motor_pwm:
  center: 1.5
  range: 0.5